### PR TITLE
#990 create persistent grafana dashboard missing steps for creating in UI

### LINKED ISSUE #990: create-persistent-grafana-dashboard missing steps for creating in UI
## Summary

https://ranchermanager.docs.rancher.com/how-to-guides/advanced-user-guides/monitoring-alerting-guides/create-persistent-grafana-dashboard

Under the "To create the ConfigMap in the Rancher UI," section, users should be instructed to add the label like the one seen in the example directly above. I have had a customer confused by it (command line did not work) and did not add the label when creating in the UI. This causes it to never import. 

It should also be noted that the user needs to log into the Grafana UI for the dashboards to import, or else they will get permission issues in the logs and it will not import.

### PR DESCRIPTION
**Pull request by martyav**
Fixes #990 

## Reminders

- See the [README](../README.md) for more details on how to work with the Rancher docs.

- Verify if changes pertain to other versions of Rancher. If they do, finalize the edits on one version of the page, then apply the edits to the other versions.

- If the pull request is dependent on an upcoming release, make sure to target the release branch instead of `main`.

## Description

From the issue:

> Under the "To create the ConfigMap in the Rancher UI," section, users should be instructed to add the label like the one seen in the example directly above. I have had a customer confused by it (command line did not work) and did not add the label when creating in the UI. This causes it to never import.

> It should also be noted that the user needs to log into the Grafana UI for the dashboards to import, or else they will get permission issues in the logs and it will not import.

### CODE REVIEW

**Comment by martyav** [comment]
Reached out to @tlatino on Slack and made some updates.

File: docs/how-to-guides/advanced-user-guides/monitoring-alerting-guides/create-persistent-grafana-dashboard.md [hunk 1]
```diff
@@ -65,19 +65,26 @@ data:
...
-1. Set up the key-value pairs similar to the example above. When entering the value for `<dashboard-name>.json`, click **Read from File** to upload the JSON data model as the value.
+1. On the **Data** tab, set up the key-value pairs similar to the example above. When entering the value for `<dashboard-name>.json`, click **Read from File** to upload the JSON data model as the value.
+1. On the **Labels & Annotations** tab, click **Add Label** and enter `grafana_dashboard` as the key, and `1` as the value.
 1. Click **Create**.
 
 **Result:** After the ConfigMap is created, it should show up on the Grafana UI and be persisted even if the Grafana pod is restarted.
 
+:::note
+
+The actual key/value pair may differ if you have modified the Helm chart to watch a different dashboard label and value.
```
Comment (btat) [inline docs/how-to-guides/advanced-user-guides/monitoring-alerting-guides/create-persistent-grafana-dashboard.md:84]:
```suggestion
The actual key-value pair may differ if you have modified the Helm chart to watch a different dashboard label and value.
```

Earlier on line 76 the hyphenated form is used.

**Review by btat — CHANGES_REQUESTED** [review]

File: versioned_docs/version-2.7/how-to-guides/advanced-user-guides/monitoring-alerting-guides/create-persistent-grafana-dashboard.md [hunk 2]
```diff
@@ -65,19 +65,26 @@ data:
...
-1. Set up the key-value pairs similar to the example above. When entering the value for `<dashboard-name>.json`, click **Read from File** to upload the JSON data model as the value.
+1. On the **Data** tab, set up the key-value pairs similar to the example above. When entering the value for `<dashboard-name>.json`, click **Read from File** to upload the JSON data model as the value.
+1. On the **Labels & Annotations** tab, click **Add Label** and enter `grafana_dashboard` as the key, and `1` as the value.
 1. Click **Create**.
 
 **Result:** After the ConfigMap is created, it should show up on the Grafana UI and be persisted even if the Grafana pod is restarted.
 
+:::note
+
+The actual key/value pair may differ if you have modified the Helm chart to watch a different dashboard label and value.
```
Comment (martyav) [inline versioned_docs/version-2.7/how-to-guides/advanced-user-guides/monitoring-alerting-guides/create-persistent-grafana-dashboard.md:84]:
```suggestion
The actual key-value pair may differ if you have modified the Helm chart to watch a different dashboard label and value.
```

File: versioned_docs/version-2.8/how-to-guides/advanced-user-guides/monitoring-alerting-guides/create-persistent-grafana-dashboard.md [hunk 3]
```diff
@@ -65,19 +65,26 @@ data:
...
-1. Set up the key-value pairs similar to the example above. When entering the value for `<dashboard-name>.json`, click **Read from File** to upload the JSON data model as the value.
+1. On the **Data** tab, set up the key-value pairs similar to the example above. When entering the value for `<dashboard-name>.json`, click **Read from File** to upload the JSON data model as the value.
+1. On the **Labels & Annotations** tab, click **Add Label** and enter `grafana_dashboard` as the key, and `1` as the value.
 1. Click **Create**.
 
 **Result:** After the ConfigMap is created, it should show up on the Grafana UI and be persisted even if the Grafana pod is restarted.
 
+:::note
+
+The actual key/value pair may differ if you have modified the Helm chart to watch a different dashboard label and value.
```
Comment (martyav) [inline versioned_docs/version-2.8/how-to-guides/advanced-user-guides/monitoring-alerting-guides/create-persistent-grafana-dashboard.md:84]:
```suggestion
The actual key-value pair may differ if you have modified the Helm chart to watch a different dashboard label and value.
```

**Review by btat — APPROVED** [review]
Approving from a docs/style/formatting perspective.

**Comment by martyav** [comment]
Merging as this got lost in the holiday shuffle -- it received a technical review already and then got approval on style grounds 2 weeks ago

**Pull request merged**